pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/PersistentTopicsBase.java

```diff
@@ -1435,14 +1437,15 @@ protected void internalGetPartitionedStats(AsyncResponse asyncResponse, boolean
                         .thenCompose(owned -> {
                             if (owned) {
                                 return getTopicReferenceAsync(partition)
-                                    .thenApply(ref ->
-                                        ref.getStats(getPreciseBacklog, subscriptionBacklogSize,
-                                            getEarliestTimeInBacklog));
+                                        .thenApply(ref ->
+                                                ref.getStats(getPreciseBacklog, subscriptionBacklogSize,
```

Comment:
i think we can add some class `XXXXOption` to encapsulate these boolean config.
if we add more boolean config it will be easy to just change `XXXXOption` without too much change the related code.  And it improves the code quality to make method signature with less parameter

Reply:
there has an GetStatsOptions ,but there has an usage somewhere. I will try to add a method